workflows/reference/reference_build.wdl

```diff
@@ -1,22 +1,26 @@
 version 1.0
 
-import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/main/workflows/reference/tasks/clinvar.wdl" as clinvar
-import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/main/workflows/reference/tasks/ensembl.wdl" as ensembl
-import "https://raw.githubusercontent.com/PacificBiosciences/pb-human-wgs-workflow-wdl/main/workflows/reference/tasks/lof.wdl" as lof
+#import "./tasks/clinvar.wdl" as clinvar
```

Comment:
In other WDLs, you uncomment the absolute URIs and comment the relative URIs.  Here you comment out both.  Is this intentional?

Reply:
Nope accident part of testing